python/GafferDispatchUI/SystemCommandUI.py

```diff
@@ -80,6 +80,27 @@
 
 			"layout:section", "Settings.Environment Variables",
 
+		),
+
+		"shell" : (
+
+			"description",
+			"""
+			When enabled, the specified command is interpreted as a shell
+			command. This allows shell semantics, such as pipes, etc. to be
+			used and generally gives the most predictable behaviour. Otherwise
```

Comment:
Omit "gives the most predictable behaviour"? It makes it sound like `shell = False` is unreliable.

Reply:
Happy with the new copy in https://github.com/GafferHQ/gaffer/commit/17f5305fb8defc8a78524fe96ca204b26d05b669?